### PR TITLE
Track individual URL timings added to docs

### PR DESCRIPTION
**Pull request by fulyauluturk**
@HSSalman can you add the two samples here https://github.com/sematext/docs/blob/master/docs/synthetics/puppeteer-scripts/ one with and one without waitForTimeout function please? And then, I'll update the links.

### CODE REVIEW

File: docs/synthetics/user-journey-scripts/track-individual-url-timings.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+await page.goto("https://www.my-company-page.com/");
+let end = new Date();
+console.log('google', start, end);
+let seconds = (end.getTime() - start.getTime()) / 1000;
+context.setMetric('googlePageLoadTime', seconds);
+```
+
+Refer to [extract page load time script](link will be added after Haris S adds the sample) for the complete example. Also, another example on how to extract page load times when using the `waitForTimeOut` can be found [here](link will be added after Haris S adds the sample).
+
+After extracting page load time metrics, you can visualize and create alert rules by following the instructions [here](https://sematext.com/docs/synthetics/metrics/#how-to-visualize-and-alert-on-metrics). This process will result in a dashboard where you can view individual page load times for each monitor run.
```
Comment (otisg) [inline docs/synthetics/user-journey-scripts/track-individual-url-timings.md:31]: After extracting page load time metrics, you can visualize ______ (performance?)

**Review by otisg — APPROVED** [review]

File: docs/synthetics/user-journey-scripts/track-individual-url-timings.md (hunk 1 above)
Comment (fulyauluturk) [reply docs/synthetics/user-journey-scripts/track-individual-url-timings.md]: After extracting page load time metrics, you can visualize them?

Comment (otisg) [reply docs/synthetics/user-journey-scripts/track-individual-url-timings.md]: +1

**Review by HSSalman — APPROVED** [review]
Tweaked a few things and added an example of how to track multiple page load times, while also using the helper functions we talked about a few weeks ago.

**Pull request merged**